### ISSUE TITLE
Add a link to the cell in the rejected message

### ISSUE THREAD

**Issue by Codain** [hotosm/osm-tasking-manager2#859]
I've received two messages saying that a cell is not complete... but I don't know which one! There is a link on the title of the message to the task but not to the cell where the issue comes from.

Note that those messages have been sent 5 monthes ago and that it may have been fixed since.

**Comment by althio** [comment]
Possible duplicate of #465

**Comment by pgiraud** [comment]
@Codain I'm surprised that you are not directed to the cell. A "task" in the tasking manager is the "cell" of a "project".
The url behind the link should be something looking like:
http://tasks.hotosm.org/project/2041#task/462

If it's not the case, then it may mean that the message has been sent to all the contributors of a project.

**Comment by bgirardot** [comment]
@Codain Yes, just to be clear the task number is probably the place you want to go to see what is up. If it was "invalidated" there will be a message on the left of the map about it.

As pierre shows above, that is the url to access a task square

**Comment by ethan-nelson** [comment]
I tested it out and, for both a task invalidation and a task mention, the subject includes the task number with a clickable link to the task in the respective project. That said, there is no text indicating the project number (in case someone wanted to navigate to the mentioned task manually)--it's only inherent in the subject-embedded url pointing to the task.
